server/data/orchestrationApiClient.test.ts

```diff
@@ -608,7 +608,7 @@ describe('orchestrationApiClient', () => {
         .matchHeader('authorization', `Bearer ${token}`)
         .reply(200, results)
 
-      const output = await orchestrationApiClient.getVisitSessions('A1234BC', prisonId, 'user1', '2')
+      const output = await orchestrationApiClient.getVisitSessions('A1234BC', prisonId, 'user1', 2)
```

Comment:
Changed type of a function parameter from `string` to `number` so quite a few minor changes like this.